nixos/modules/services/monitoring/gatus.nix

```
@@ -0,0 +1,147 @@
+{ pkgs, lib, config, ... }:
+let
+  cfg = config.services.gatus;
+
+  settingsFormat = pkgs.formats.yaml { };
+
+  inherit (lib)
+    getExe
+    literalExpression
+    mdDoc
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOptionMD
+    types
+    ;
+in
+{
+  options = {
+    services.gatus = {
+      enable = mkEnableOption (mdDoc "Gatus");
+
+      package = mkPackageOptionMD pkgs "gatus" { };
```

I think you can replace it with: 

```suggestion
      package = mkPackageOption pkgs "gatus" { };
```